### DISCUSSION TITLE
Jupyter Notebook: Display error in .ipynb file viewer for strings containing HTML tags

### DISCUSSION THREAD

**Discussion by GermanWarez** [community/community#40144] [Projects and Issues]
### Select Topic Area

Bug

### Body

Examples of the bug:
`a='<alert>foobar</alert>'`  shows  as `a=foobar'`

`a='<h2foobar'`  shows  as `a='`

`a='<h2[^>]foobar'`  shows as
a='
]foobar
(formatting of ']foobar' is h2)

**Comment by ossanna16** [comment]
Hi there @GermanWarez and welcome to our community! Thank you for reporting this bug!
